CHANGELOG.md

```diff
@@ -11,12 +11,13 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Added
 
 - New metric `sidecar.points.produced` counts total points produced from the WAL. (#187)
+- `sidecar.metrics.invalid` includes explanation for all unreported metrics, labeled by `key_reason` and `metric_name`, now covers filtered points, metadata errors, and explanations from the server. (#191)
```

Comment:
we should have a note about `sidecar.metrics.invalid` to the readme, im not seeing one currently

Reply:
It's `sidecar.metrics.failing`, the changelog had a bug. Fixed.
The README had a note about validation errors, I've added one about local metadata errors.